Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+    )
+
+
+def jira_asset_object_search_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Searches for Jira asset objects.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'ql_query': The query string to search for.
+        - 'include_attributes': Whether to include the object's attributes in the response.
+        - 'page': The page number to retrieve.
+        - 'page_size': The number of objects to retrieve per page.
+        - 'limit': The maximum number of objects to retrieve.
+    :return: A CommandResults object containing the objects as output, and a human-readable markdown table.
+    """
+    # build request params
+    ql_query = args.get('ql_query')
+    include_attributes = argToBoolean(args.get('include_attributes', False))
+    page = int(args.get('page', 1))
+    page_size = int(args.get('page_size', 50))
+    limit = args.get('limit')
+
+    # build outputs
+    res = client.search_objects(ql_query, include_attributes, page, page_size, limit)
+    pascal_res = convert_keys_to_pascal(res['objectEntries'], {'id': 'ID'})
+    hr_headers = ['ID', 'Label', 'Type', 'ObjectKey']
+    outputs, readable_output = get_object_outputs(pascal_res)
+
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_OUTPUTS_BASE_PATH}.Object',
+        outputs_key_field='ID',
+        outputs=outputs,
+        readable_output=tableToMarkdown('Object', readable_output, headers=hr_headers, removeNull=True)
+    )
+
+
+def jira_asset_attribute_json_create_command(client: Client, args: Dict[str, Any]) -> [dict, CommandResults]:
+    """
+    Creates a Jira asset attribute JSON object.
+    :param client: Client object for performing API requests.
+    :param args: A dictionary of command arguments:
+        - 'object_type_id': The ID of the object type to create the attribute JSON for.
+        - 'is_required': Whether to include only required attributes in the response.
+        - 'is_editable': Whether to include only editable attributes in the response.
+    :return: A CommandResults object containing the attribute JSON as a file entry, and a human-readable json string.
+    """
+    # build request params
+    object_type_id = args.get('object_type_id')
+    is_editable = args.get('is_editable', False)
+
+    # build outputs
+    res = client.get_object_type_attributes(object_type_id=object_type_id, is_editable=is_editable)
+    if args.get('is_required'):
+        res = [attribute for attribute in res if attribute.get('minimumCardinality') > 0]
+
+    outputs = {'attributes': [{
+        "objectTypeAttributeId": attribute.get("id"),
+        "objectAttributeValues": [{"value": ''}]
+    } for attribute in res]}
+
+    hr_command_results = CommandResults(readable_output=json.dumps(outputs))
+    file_entry = fileResult(filename='attributes.json', data=json.dumps(outputs, indent=2), file_type=EntryType.ENTRY_INFO_FILE)
```

Comment:
why return this as an info file instead of to the context?

Reply:
Its a helper function. The returned structure has no real meaning or use, its just an empty structure for the user to fill out. We return it both in the HR and in a info file format so the user could have an easier time filling in the values.